Gemfile

```diff
@@ -3,9 +3,9 @@ source "https://rubygems.org"
 gemspec
 
 gem "rake"
-gem "redis-namespace", github: "resque/redis-namespace", branch: :master
+gem "redis-namespace", "1.11.0"
```

Comment:
This is the version that `platform` is targeting to upgrade to
- https://github.com/Appboy/platform/pull/76763